packages/backend/discovery/parallel/ethereum/config.jsonc

```diff
@@ -18,7 +18,8 @@
     "0xD368b8dC5cB6fA26A53b7588db9A87E509A72d89": "UpgradeExecutor",
     "0x6Eb9240d4add111D5Fc81b10Ff12eECabcf9752d": "L1ERC20Gateway",
     "0xa1c86E2362dba0525075622af6d5f739B1304D45": "L1CustomGateway",
-    "0x150286BdbE7C8Cd23D41a8e1e64438e0dc04dc3d": "L1WETHGateway"
+    "0x150286BdbE7C8Cd23D41a8e1e64438e0dc04dc3d": "L1WETHGateway",
+    "0x4a4962275DF8C60a80d3a25faEc5AA7De116A746": "GnosisSafe"
```

Comment:
please rename it to `OwnerMultisig` as i did here, to prevent me from missing this merge conflict later :p https://github.com/l2beat/l2beat/pull/2550/files

Reply:
🏁 
[chore: rename parallel discovered multisig](https://github.com/l2beat/l2beat/pull/2600/commits/435e5861677f05728b7e2216bc16e740fe79b0ac)